### PR TITLE
Lesson 5.1

### PR DESCRIPTION
**Pull request by SokolovskayaE**
Debugging

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,66 +25,90 @@ function getRandomNumber(min, max) {
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  //attempts = attempts + 1;
 
   hideAllMessages();
 
+  if (guess <= 0) {
```
Comment (toma63) [inline index.js:40]: Should also check for integer here.  Currently accepts floating point numbers.

**Review by toma63 — APPROVED** [review]
Nice work!  Works as specified and enough bugs found.  Found one more bug which could be fixed (floating point numbers).

File: index.js (hunk 1 above)
Comment (SokolovskayaE) [reply index.js:40]: As I understand it need to be like this: if (guess <= 0 && Number.isInteger(guess)) {
Thanks a Lot!

Comment (toma63) [reply index.js:40]: guess should be > 0, < 100 and number.isInteger() must be true.

**Pull request merged**